packages/cli/src/commands/init/init.js

```diff
@@ -1,100 +1,115 @@
-/**
- * Copyright (c) Facebook, Inc. and its affiliates.
- *
- * This source code is licensed under the MIT license found in the
- * LICENSE file in the root directory of this source tree.
- *
- * @format
- */
-
-import fs from 'fs';
+// @flow
+import fs from 'fs-extra';
 import minimist from 'minimist';
-import path from 'path';
-import process from 'process';
-import printRunInstructions from '../../tools/generator/printRunInstructions';
-import {createProjectFromTemplate} from '../../tools/generator/templates';
-import * as PackageManager from '../../tools/PackageManager';
+import semver from 'semver';
+import type {ContextT} from '../../tools/types.flow';
+import {validateProjectName} from './validate';
+import DirectoryAlreadyExistsError from './errors/DirectoryAlreadyExistsError';
+import printRunInstructions from './printRunInstructions';
 import logger from '../../tools/logger';
+import {
+  installTemplatePackage,
+  getTemplateConfig,
+  copyTemplate,
+  executePostInitScript,
+} from './template';
+import {changePlaceholderInTemplate} from './editTemplate';
+import * as PackageManager from '../../tools/PackageManager';
```

Comment:
Do we use `PackageManager` anywhere outside of this command? Maybe we can move it here. Just a "misc", but I am trying to remove most of the "tools" as they are not useful except for the really particular use-case a command might be serving.

Reply:
Also, should be probably lower-case since it's not a "class"?

Reply:
Let's do it in a separate PR

Reply:
We use it in a bunch of other placed, e.g. install/uninstall commands (feel free to grep for this)